### PR TITLE
MRG: Fix raw PSD

### PR DESCRIPTION
**Pull request by Eric89GXL**
Does two things:

1. Adds `SEEG` and `ECoG` to `raw.plot_psd`, which formely just did mag/grad/eeg.

2. Allows for a list of axes to be passed when `picks=None`.

Ready for review/merge from my end.

### CODE REVIEW

File: mne/viz/raw.py [hunk 1]
```diff
@@ -401,21 +401,29 @@ def _set_psd_plot_params(info, proj, picks, ax, area_mode):
...
+        seegs = [False, False, False, True, False]
+        ecogs = [False, False, False, False, True]
+        names = ['Magnetometers', 'Gradiometers', 'EEG', 'SEEG', 'ECoG']
         picks_list = list()
         titles_list = list()
-        for meg, eeg, name in zip(megs, eegs, names):
-            picks = pick_types(info, meg=meg, eeg=eeg, ref_meg=False)
+        for meg, eeg, seeg, ecog, name in zip(megs, eegs, seegs, ecogs, names):
+            picks = pick_types(info, meg=meg, eeg=eeg, seeg=seeg, ecog=ecog,
+                               ref_meg=False)
```
Comment (kingjr) [inline mne/viz/raw.py]: This lookup table would probably benefit from being in `mne/channels`: I've been confronted to similar loop picking between meg/eeg/seeg at least twice in the `tests/`.

I would probably make a `dict` of `dict` e.g. `ch_types = dict(eegs=dict(eeg=True, meg=False ...))`

File: mne/viz/raw.py [hunk 2]
```diff
@@ -401,21 +401,29 @@ def _set_psd_plot_params(info, proj, picks, ax, area_mode):
...
         if len(picks_list) == 0:
-            raise RuntimeError('No MEG or EEG channels found')
+            raise RuntimeError('No data channels found')
+        if ax is not None:
+            if isinstance(ax, plt.Axes):
+                ax = [ax]
+            if len(ax) != len(picks_list):
+                raise ValueError('For this dataset with picks=None %s axes '
+                                 'must be supplied, got %s'
+                                 % (len(picks_list), len(ax)))
```
Comment (kingjr) [inline mne/viz/raw.py]: is `picks` necessarily `None` here?

**Comment by kingjr** [comment]
LGTM

File: mne/viz/raw.py (hunk 2 above)
Comment (Eric89GXL) [inline mne/viz/raw.py]: Yes check conditional a few lines earlier

**Comment by agramfort** [comment]
thx @Eric89GXL

**Pull request merged**